content/download/index.html.haml

```diff
@@ -258,6 +258,14 @@ title: Jenkins download and deployment
   %p{:style=>'margin-bottom:1cm;margin-top:.2cm;'}
     Jenkins Reference Architecture and one-click deployment on Oracle Cloud Infrastructure
 
+%div
+  %a{:href=>'https://github.com/civo/kubernetes-marketplace/tree/master/jenkins'}
+    %img{:src=>'/images/civo.png', :width=>'320'}
+  %p{:style=>'margin-bottom:1cm;margin-top:.2cm;'}
+    Jenkins one-click deployment on
+      %a.item{:href=>'https://www.civo.com/'}
```

Comment:
How awkward that I used the wrong indentation in an indentation significant file.
```suggestion
    %a.item{:href=>'https://www.civo.com/'}
```